--- ../www/lib/users.php	2015-11-17 18:51:35.832013944 +0000
+++ ../www/lib/users.php	2015-11-17 14:57:30.389932225 +0000
@@ -293,7 +293,7 @@
 	public function getById($id)
 	{			
 		$db = new DB();
-		$sql = sprintf("select users.*, userroles.name as rolename, userroles.hideads, userroles.canpreview, userroles.canpre, userroles.apirequests, userroles.downloadrequests, NOW() as now from users inner join userroles on userroles.ID = users.role where users.id = %d ", $id);
+		$sql = sprintf("select users.*, userroles.name as rolename, userroles.*, NOW() as now from users inner join userroles on userroles.ID = users.role where users.id = %d ", $id);
 		return $db->queryOneRow($sql);		
 	}	
 	
